From 22f8fb66378d37cd6612660d3c204853e983e6b7 Mon Sep 17 00:00:00 2001
From: Jin Qian <jinqian@google.com>
Date: Tue, 10 Jan 2017 16:11:07 -0800
Subject: [PATCH] ANDROID: uid_sys_stats: rename uid_cputime.c to
 uid_sys_stats.c

This module tracks cputime and io stats.

Signed-off-by: Jin Qian <jinqian@google.com>
Bug: 34198239
Change-Id: I9ee7d9e915431e0bb714b36b5a2282e1fdcc7342
(cherry picked from commit c876f27f83eaae61afb1dc8d585e6e627ed4d8bb)
Signed-off-by: Guenter Roeck <groeck@chromium.org>

[rebase52(rrangel): Context conflict]
Signed-off-by: Raul E Rangel <rrangel@chromium.org>
---
 drivers/misc/Kconfig                            | 7 ++++---
 drivers/misc/Makefile                           | 2 +-
 drivers/misc/{uid_cputime.c => uid_sys_stats.c} | 0
 3 files changed, 5 insertions(+), 4 deletions(-)
 rename drivers/misc/{uid_cputime.c => uid_sys_stats.c} (100%)

diff --git a/drivers/misc/Kconfig b/drivers/misc/Kconfig
index f8358ebdccd4..c2e488c81a7d 100644
--- a/drivers/misc/Kconfig
+++ b/drivers/misc/Kconfig
@@ -487,12 +487,13 @@ config PVPANIC
 	  a paravirtualized device provided by QEMU; it lets a virtual machine
 	  (guest) communicate panic events to the host.
 
-config UID_CPUTIME
-	bool "Per-UID cpu time statistics"
+config UID_SYS_STATS
+	bool "Per-UID statistics"
 	depends on PROFILING
 	help
 	  Per UID based cpu time statistics exported to /proc/uid_cputime
-
+	  Per UID based io statistics exported to /proc/uid_io
+	  Per UID based procstat control in /proc/uid_procstat
 
 source "drivers/misc/c2port/Kconfig"
 source "drivers/misc/eeprom/Kconfig"
diff --git a/drivers/misc/Makefile b/drivers/misc/Makefile
index 2d5f1f308833..8becbae82fe0 100644
--- a/drivers/misc/Makefile
+++ b/drivers/misc/Makefile
@@ -56,7 +56,7 @@ obj-$(CONFIG_CXL_BASE)		+= cxl/
 obj-$(CONFIG_PCI_ENDPOINT_TEST)	+= pci_endpoint_test.o
 obj-$(CONFIG_OCXL)		+= ocxl/
 obj-y				+= cardreader/
-obj-$(CONFIG_UID_CPUTIME)	+= uid_cputime.o
+obj-$(CONFIG_UID_SYS_STATS)	+= uid_sys_stats.o
 obj-$(CONFIG_PVPANIC)   	+= pvpanic.o
 obj-$(CONFIG_HABANA_AI)		+= habanalabs/
 obj-$(CONFIG_XILINX_SDFEC)	+= xilinx_sdfec.o
diff --git a/drivers/misc/uid_cputime.c b/drivers/misc/uid_sys_stats.c
similarity index 100%
rename from drivers/misc/uid_cputime.c
rename to drivers/misc/uid_sys_stats.c
-- 
2.23.0.rc1.153.gdeed80330f-goog

